src/js-marketing-notification.ts

```diff
@@ -1,32 +1,36 @@
 import { LocalStorageManager } from './github/gtmsportswear/js-local-storage-manager@1.0.2/local-storage-manager';
 
+export interface NotificationNode {
+  classNames?: Array<string>,
```

Comment:
I see. Yeah. I think those might be safe to assume these can be styled without any special css classes.